shared/teams/team/members-tab/member-row/index.tsx

```diff
@@ -73,28 +73,28 @@ export const TeamMemberRow = (props: Props) => {
   }
 
   return (
-    <Box style={active ? stylesContainer : stylesContainerReset}>
+    <Box style={Styles.collapseStyles([styles.container, active && styles.containerReset])}>
```

Comment:
```suggestion
    <Box style={active ? styles.container : styles.containerReset}>
```

Reply:
note these are not equivalent. can only be if containerReset is a super set in the style but not in general

Reply:
It is the ternary in master, the style on the rows is weird with `collapseStyles` (they're blue)

Reply:
k. i think the ternary is better, but it could also be `[!active && styles.container, active && styles.containerReset]` etc if mixing a bunch of concepts

Reply:
Note that `containerReset` isn't a superset of `container` here anymore. I changed it so it only includes the difference now.

Reply:
It should be `!active && styles.containerReset`

Reply:
oops; good catch!!